.github/workflows/test.yml

```diff
@@ -31,3 +31,8 @@ jobs:
         outputPath: 'IssuesReport.md'
         configPath: 'test/issues-report.json'
         token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/upload-artifact@v2
+      with:
+        # This file gets produced by main.test.ts
```

Comment:
For this job, `IssuesReport.md` will be produced by the action itself, right?

Reply:
yes, I'll update the comment, thanks :-)